package.json

```diff
@@ -112,6 +113,7 @@
 		"eslint-plugin-wpcalypso": "5.0.0",
 		"glob": "7.1.6",
 		"husky": "4.3.8",
+		"jetpack-cli": "file:tools/cli",
```

Comment:
Ran into an odd failure case here.

1. `yarn install`
2. ✔️ Command `yarn jetpack` works.
3. Make some changes in tools/cli
4. ❌ Changes don't show up.
5. `yarn install` again.
6. ❌ Changes still don't show up.
7. `yarn install --force`
8. ✔️ Now the changes show up.

Reply:
Ah, yes. I should have added instructions on how to development the CLI.

With the `tools/cli` folder, run `yarn link`. Then in the repo root run `yarn link jetpack-cli`.

Without that step, yarn will actually just copy the files into `node_modules`. For me, it only seemed to matter when making changes to the CLI. If bringing in a patch and installing, it seems to work as expected.

Reply:
Can we have it do that automatically somehow?

Reply:
Updated the description.

Reply:
There doesn't appear to be a straight-forward way for Yarn to do this automatically. I'm playing with some scripts to see if I can do it directly. Worst case, should be able to do it via the CLI.

Reply:
Softened the check in 9a8989caa .

In 4807803fb, I added a postinstall script that would run a CLI command to link up the projects, so this can be considered done.